compose_files/pulp/docker-compose.yml

```diff
@@ -22,6 +22,7 @@ services:
 
   migration_service:
     image: "quay.io/redhat-services-prod/pulp-services-tenant/pulp:latest"
+    platform: linux/amd64
```

Comment:
i don't think we want to push the changes here in case it messes up anyone else, i'm not sure if it would but just in case. maybe we can ask someone using fedora to test if this breaks anything and add this to another PR if not, it would be nice to have for us mac users :)

Reply:
i can test it

Reply:
works fine, ack on this change

Reply:
awesome!! 🎉 thank you for testing!